connection_go18.go

```diff
@@ -0,0 +1,11 @@
+// +build go1.8
+
+package mysql
+
+import (
+	"context"
+)
+
+func (mc *mysqlConn) Ping(ctx context.Context) error {
+	return mc.writeCommandPacket(comPing)
```

Comment:
https://dev.mysql.com/doc/internals/en/com-ping.html

You should read `OK` packet after send COM_PING.
Additionally, cancellation should be implemented too.
When Ping is cancelled between PING and OK, the connection should be disposed.